lib/srv/db/dynamodb/engine.go

```diff
@@ -30,10 +30,10 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/aws/aws-sdk-go/aws/endpoints"
-	"github.com/aws/aws-sdk-go/service/dax"
-	"github.com/aws/aws-sdk-go/service/dynamodb"
-	"github.com/aws/aws-sdk-go/service/dynamodbstreams"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/dax"
```

Comment:
The dax APIs sourced here are only for deployment, not for interacting with the "database". 

I don't think db access works with dax today: https://github.com/aws/aws-dax-go

I propose we drop dax here and track dax separately. WDYT

Reply:
It's easier and cleaner without the dax stuff now. I doubt anyone is using it today through database access. 

on the other hand, if you prefer to keep it now, we don't have to drop it later.